e2e_test/batch/catalog/sysinfo.slt.part

```diff
@@ -20,7 +20,7 @@ statement ok
 create index tab_idx on tab(num desc);
 
 query T
-select pg_get_indexdef('tab_idx'::regclass);
+select pg_get_indexdef(('"' || current_database() || '".' || 'public.tab_idx')::regclass);
 ----
 CREATE INDEX tab_idx ON tab(num DESC)
```

Comment:
Instead of modifying the original test case, shall we add a new one?

Reply:
Sure.